src/backend/marsha/core/serializers/video.py

```diff
@@ -530,19 +530,13 @@ def get_live_info(self, obj):
             For other users, an empty dictionnary is returned.
         """
         can_return_live_info = self.context.get("can_return_live_info", False)
+        live_info = {}
 
-        if obj.live_state is None or can_return_live_info is False:
-            return {}
-
-        live_info = {
-            "medialive": {
-                "input": {
-                    "endpoints": obj.live_info["medialive"]["input"]["endpoints"],
-                }
-            },
-        }
-
-        if settings.JITSI_ENABLED and obj.live_type == JITSI:
+        if (
```

Comment:
Security wise, the jitsi info should be contain a personal token to restrict access to the person authorized by the instructor... and should therefore only be given at the time the access is granted and not on first access to the video!
I know access tokens are not yet there but as they are coming soon, I think you shouldn't modify this view and implement a way for an instructor to get an authorization token for a given student and pass it to him/her (via xmpp?) upon request?

Reply:
Yes I should send jitsi info in the message accepting to join the discussion.

Reply:
Done